src/pages/sso/sso.ts

```diff
@@ -5,12 +5,14 @@ import { ToastController } from 'ionic-angular';
 
 @Component({
     selector: 'page-sso',
-    templateUrl: 'sso.html'
+    templateUrl: '../underConstruction/construction.html'
```

Comment:
@ciaranRoche is this the correct template?

Reply:
nice spot :+1: